ci/nodepool/scripts/prepare_pulp3_node.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+HOSTNAME="$1"
+DOCKER="$(grep docker 2>/dev/null <<< ${HOSTNAME})"
+
+echo "Disable selinux"
+sudo setenforce 0
+
+# Fail immediately on error
+set -ex
```

Comment:
This `-e` will cause bash to fail if an error occurs. But errors inside pipelines are ignored. Use `-o pipefail` for that. References to unset variables are also allowed. Use `-u` for that. I recommend the following:

```sh
set -euo pipefail
```

Or, if you want debugging:

```sh
set -xeuo pipefail
```

Reply:
Also, why not move this to the top of the script? I see no reason perform variable  assignments in an unsafe manner.